agent/util-scripts/gold/test-client-tool-meister/test-61.txt

```diff
@@ -0,0 +1,291 @@
++++ Running test-61 test-client-tool-meister lite with-remotes
+"mpstat" tool is now registered for host "testhost.example.com" in group "lite"
+"dcgm" tool is now registered for host "testhost.example.com" in group "lite"
+"mpstat" tool is now registered for host "remote-a.example.com" in group "lite"
+"mpstat" tool is now registered for host "remote-b.example.com", with label "blue", in group "lite"
+"node-exporter" tool is now registered for host "remote-b.example.com", with label "blue", in group "lite"
+"dcgm" tool is now registered for host "remote-c.example.com", with label "red", in group "lite"
+"pcp" tool is now registered for host "remote-c.example.com", with label "red", in group "lite"
+/var/tmp/pbench-test-utils/opt/pbench-agent/util-scripts/pbench-tool-meister-start:110: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
```

Comment:
This error message is coming from using Python 3.11.  It does not show up when using Python 3.9.

We need to find a suitable replacement for `find_executable`.

Filed issue https://github.com/distributed-system-analysis/pbench/issues/2724 to track.